Snowball-Finance/axial-frontend
# Swap Approval UI integration

To test:
1. Please check the design as per figma
2. Please check the flow of approval in swap page
3. Approval steps
  a) Input the proper amount of tokens
  b) Click on the swap button
  c) Review swap will open, with approve button enable(if token isn't approved), disabled the swap button when approve is enable
  d) After approve button clicks, approving component render, please check the design of it as well
  e) After approval, swap button will enable and user can perform swapping
   f) After successful swap, transaction status component render with transaction link of snowtrace.io

Asana ticket: https://app.asana.com/0/1201433089668827/1202064151842525
